vue-app/src/components/auth-login.vue

```diff
@@ -14,10 +14,15 @@ export default {
       const url = `/.auth/login/${this.provider}?${redirect}`;
       window.location.href = url;
     },
+    handleKeyDown() {
```

Comment:
added due to lint rule stating that non button element with click handlers need to have keyhandler for accessibility, I can disable this rule or keep this code, please let me know which @johnpapa